server/routes/address.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const db = require('../../db/models');
+const Address = db.Address;
+
+const router = express.Router();
+
+router.get('/:type/user/:userId', (req, res, next) => {
+    Address.findOne({
+                    where: { addressType: req.params.type, 
+                            status: 'active', 
+                            userId: req.params.userId }})
+    .then((address) => res.json(address))
+    .catch(next);
+});
+
+router.post('/:type/user/:userId', (req, res, next) => {
+    let argument = {
+        addressType: req.params.type,
```

Comment:
Please also pass userId: req.params.userId with the post. Otherwise everything looks good.

Reply:
Whoops..thanks for catching that.  Fixed.